docs/README.md

```diff
@@ -36,7 +36,7 @@ Give Jenkins read access to Secrets Manager with an IAM policy.
 
 Required permissions:
 
```

Comment:
Can you add the AWS Policy in our discussion issues? That one works.

Reply:
Added an example policy, let me know if it works and makes sense

Reply:
Not sure if you need to add `resource *` for list secret, since that is enforced for list secret perm.

Reply:
The AWS docs say that `ListSecrets` does not have a `Resource` condition: https://docs.aws.amazon.com/service-authorization/latest/reference/list_awssecretsmanager.html

So we should be okay to omit it